### DISCUSSION TITLE
Issue with simulation time not changing in "verif_magnoh" problem

### DISCUSSION THREAD

**Discussion by lyb-bbb** [PrincetonUniversity/athena#526] [General]
Hello everyone
  I'm currently testing the verification code for the "verif_magnoh" problem in the Beresnyak_pop_2022 repository, specifically for "simulating a pulsed-power-driven plasma with ideal MHD". However, I've encountered an issue where the simulation time (t) is not changing during the execution of the code, and the time step (dt) is consistently reported as 0. 

  The configure command is "./configure.py -b --flux hlle --coord cylindrical --prob verif_magnoh   -hdf5 -mpi ", and I just change the output filetype to hdf5. 

  Any guidance or assistance in resolving this issue would be greatly appreciated. Thank you!

![output](https://github.com/PrincetonUniversity/athena/assets/91472133/7793a8f1-84a1-4914-a4f8-75df541dd745)

**Comment by c-white** [comment]
Is this related to https://github.com/beresnyak/verif_coupling ?

If so, that's a user's personal version of the code. In particular, `verif_magnoh.cpp` is not a part of this repository, so you probably need to ask for help over there rather than here.

**Reply by lyb-bbb** [reply]
Thanks for your reply. I'm new in athena so I'm worried that's a common problem due to my wrong setting， 

And I wonder in which file I can check the calculation of "dt"

**Reply by felker** [reply]
https://github.com/PrincetonUniversity/athena/blob/master/src/hydro/new_blockdt.cpp
